package.json

```diff
@@ -6,7 +6,7 @@
   "main": "index.js",
   "scripts": {
     "install": "napa 'eligrey/FileSaver.js:file-saver' 'd3/d3-plugins:d3-plugins'",
-    "postinstall": "cd ./node_modules/babbage.ui/node_modules/ && rm -rf c3 d3 bubbletree raphael pivottable/node_modules/jquery",
+    "postinstall": "echo '(cd ./node_modules/babbage.ui/node_modules/ && rm -rf c3 d3 bubbletree raphael pivottable/node_modules/jquery) || true'",
```

Comment:
Good catch... for some reason this line prevented `npm install` to work.
I'm trying a clean install now to see if it reproduces.